### ISSUE TITLE
How to add new Language? Namely Portuguese

### ISSUE THREAD

**Issue by tupaschoal** [inaka/sheldon#57]
I see that the only existing dictionary is English, but I'd like to use it for Portuguese-BR. Seems like it would only need a ```bazinga.txt``` and a ```dictionary.txt```, correct?

If I can provide these, would you merge the PR?

Thanks!

**Comment by ferigis** [comment]
no. Every time you got an error from sheldon he will send you a funny message (bazinga), that file just have a list with bazingas (a bazinga per line)

**Comment by tupaschoal** [comment]
Ah, I see. I can translate it myself, I'm just looking for a open source dictionary for portuguese to add along with the PR.
